arapp.json

```diff
@@ -17,12 +17,22 @@
       "appName": "placeholder-app-name.open.aragonpm.eth"
     },
     "rinkeby": {
+      "apm": {
+         "ipfs": {
+           "gateway": "https://ipfs.eth.aragon.network/ipfs"
```

Comment:
I'm not sure about this change...

So `apm publish` will upload the artifacts to the default rpc: `http://localhost:5001`
And then in the last step (`Fetch published repo`), it will query this gateway for the artifacts.

Because the `ipfs.eth.aragon.network` node will need to fetch it from the local node, this might become very slow or even fail (if it takes more than `30m`). :thinking: 	

I just tried publishing this and the last step took 5 minutes.

The only benefit of adding this is that it propagates the content to the a1 gateway, but at the same time I feel like commands should be fast and not do too many things.

I think the `apm.ipfs.gateway` option is only useful for power users interacting with the dao commands, so they don't have to start their own ipfs node and worry about having the files locally, and not so much for developers that want to publish, for propagation they have the `ipfs propagate` command.

Reply:
I thought It would be great to propagate as well when publish to Rinkeby or Mainnet. But it's a fair point to not include too much in one single command 👍

Reply:
If we wanted to propagate automatically, we should probably run the `aragon ipfs propagate` command on the published command rather than trying to fetch it from a specific gateway anyway.

Reply:
Yes, let's out for this instead: https://github.com/aragon/aragon-cli/issues/535